build/tools/amebad/gnu_utility/README.md

```diff
@@ -0,0 +1,13 @@
+# AmebaD Jlink Debug Function
+
+### Requirements
```

Comment:
If specific version is not needed, I think we can remove this.

Reply:
Ok, I will remove it, thanks!